Formula/lib/libgnt.rb

```diff
@@ -1,14 +1,13 @@
 class Libgnt < Formula
   desc "NCurses toolkit for creating text-mode graphical user interfaces"
   homepage "https://keep.imfreedom.org/libgnt/libgnt"
-  url "https://downloads.sourceforge.net/project/pidgin/libgnt/2.14.3/libgnt-2.14.3.tar.xz"
-  sha256 "57f5457f72999d0bb1a139a37f2746ec1b5a02c094f2710a339d8bcea4236123"
+  url "https://keep.imfreedom.org/libgnt/libgnt/archive/v2.14.4.tar.gz"
+  sha256 "40840dd69832fdae930461e95d8505dabc8854b98a02c258c8833e0519eabe69"
   license "GPL-2.0-or-later"
-  revision 1
 
   livecheck do
-    url "https://sourceforge.net/projects/pidgin/rss?path=/libgnt"
-    regex(%r{url=.*?/libgnt[._-]v?(\d+(?:\.\d+)+)\.t}i)
+    url "https://keep.imfreedom.org/libgnt/libgnt/tags"
+    regex(%r{href=["']/libgnt/libgnt/rev/v?(\d+(?:\.\d+)+)["' >]}i)
```

Comment:
```suggestion
    regex(%r{href=["']?[^"' >]*?/rev/v?(\d+(?:\.\d+)+)["' >]}i)
```
To be consistent with other formulae, like `renovate`.